profiles/tests/test_profiles_graphql_gdpr_api.py

```diff
@@ -443,8 +443,8 @@ def get_response(request, context):
 
     assert service1_mocker.call_count == 1
     assert service2_mocker.call_count == 1
-    assert "dry_run=True" in service1_mocker.request_history[0].text
-    assert "dry_run=True" in service2_mocker.request_history[0].text
+    assert "dry_run=true" in service1_mocker.request_history[0].url
+    assert "dry_run=true" in service2_mocker.request_history[0].url
```

Comment:
The request_history objects have both `qs` and `query` attributes which could be more appropriate for these asserts. Applies also to a few more similar places in this PR.